dependencies-in-practice.Rmd

```diff
@@ -656,7 +656,7 @@ count <- function(x, ..., wt = NULL, sort = FALSE, name = NULL) {
   UseMethod("count")
 }
 
-#' @export
+#' @exportS3method
```

Comment:
I think that this can still be `@export`? I'm not sure I'm yet ready to say that every S3 method should use `@exportS3method`.

Reply:
```suggestion
#' @export
```